Avoid crashing ospfd if sent a route delete for an unknown route.

diff -Naur quagga-0.99.24/ospfd/ospf_asbr.c quagga-0.99.24.new/ospfd/ospf_asbr.c
--- quagga-0.99.24/ospfd/ospf_asbr.c	2015-03-02 15:50:57.000000000 -0800
+++ quagga-0.99.24.new/ospfd/ospf_asbr.c	2016-01-18 22:53:38.106791598 -0800
@@ -178,6 +178,11 @@
 ospf_external_info_delete (u_char type, struct prefix_ipv4 p)
 {
   struct route_node *rn;
+  struct route_table *table = EXTERNAL_INFO (type);
+
+  if (!table) {
+	  return;
+  }
 
   rn = route_node_lookup (EXTERNAL_INFO (type), (struct prefix *) &p);
   if (rn)
